test/extensions/upstreams/tcp/generic/config_test.cc

```diff
@@ -24,11 +25,18 @@ namespace Extensions {
 namespace Upstreams {
 namespace Tcp {
 namespace Generic {
-class TcpConnPoolTest : public ::testing::Test {
+class TcpConnPoolTest : public testing::TestWithParam<bool> {
 public:
   TcpConnPoolTest() {
     EXPECT_CALL(lb_context_, downstreamConnection()).WillRepeatedly(Return(&connection_));
   }
+
+  void setup() {
```

Comment:
void SetUp() override {
and it'll get called automatically by the google test code :-)
/wait